File: C/Chapter 4 - Functions/2K221010/patteren_number1_ascending.c
//Loops:

//Pattern Programs

/*

Ex. Number Pattern - Ascending

1
1 2
1 2 3
1 2 3 4
1 2 3 4 5

*/

#include<stdio.h>
void main()
{
    int count,i,j,num;

    printf("\nEnter the count: ");
    scanf("%d",&count);
    printf("\n");


    for(i=1;i<=count;i++)
        {
            for(j=1;j<=i;j++)
            {
                printf("%d ",j);
            }
            printf("\n");
        }

}

File: Back End/Python/Chapter 05 - Control Statements/2K221229/patteren_number1_ascending.py
# //Loops:
#
# //Pattern Programs
#
# /*
#
# Ex. Number Pattern - Ascending
#
# 1
# 1 2
# 1 2 3
# 1 2 3 4
# 1 2 3 4 5
#
# */

count = int(input("Enter the count: "))

for i in range(0, count + 1, 1):
    for j in range(1, i + 1, 1):
        print(j, ' ', end='')
    print(end='\n')
